5-April-22/upgrade.md

```diff
@@ -0,0 +1,94 @@
+# Juno April 5, 2022 Chain Halt Upgrade
+This upgrade is effectively a hard fork with the same chain-id. What this means, is we'll need to delete all previous data and start from a new genesis.
+
+NOTE: This assumes you've already ran through setting up a juno node here: https://docs.junonetwork.io/validators/getting-setup
+
+## Performing the  Upgrade
+
+### 1. Stop the node
+```
+sudo systemctl stop junod
+```
+
+### 2. Backup ~/.juno/config/priv_validator_key.json
+This is only relevant if you're upgrading your validator. Your priv_validator_key.json is how your validator is identified. If you haven't backed it up already, *DO SO NOW*.
+
+An example method for doing so is as follows, which will copy it to the home dir:
+```sh
+cd ~
+cp ~/.juno/config/priv_validato_state.json .
+```
+
+### 3. Delete previous chain state
+Because we'll be starting from a new genesis, the previous data is no longer necessary. Similarly, the priv_validator_state.json is not necessary because there is no previous block to sign from.
+
```

Comment:
backup priv_validator_state in the event this ipgrade does not work, it may be needed for an alternate solution.

Reply:
Yes.